bolt-sidecar/src/api/commitments/handlers.rs

```diff
@@ -43,9 +73,12 @@ pub async fn rpc_entrypoint(
         })),
 
         GET_METADATA_METHOD => {
+            let metadata: MetadataResponse =
+                (api.limits(), BOLT_SIDECAR_VERSION.to_string()).into();
```

Comment:
This doesn't feel very idiomatic, does the following work?
```rs
let metadata = MetadataResponse {
   version: BOLT_SIDECAR_VERSION.to_string(),
   ..api.limits()
}
```

Reply:
No, because `api.limits()` returns `LimitOpts` and not `MetadataResponse`

Reply:
ah right, mb